pydantic_settings/main.py

```diff
@@ -214,11 +223,12 @@ def _settings_build_values(
         sources = self.settings_customise_sources(
             self.__class__,
             init_settings=init_settings,
-            cli_settings=cli_settings,
             env_settings=env_settings,
             dotenv_settings=dotenv_settings,
             file_secret_settings=file_secret_settings,
         )
+        if _cli_parse_args:
+            sources = (cli_settings,) + sources
```

Comment:
If `_cli_parse_args` is provided, it will always be topmost source.

Reply:
Shouldn't it be `if cli_parse_args`?

I mean to enable it by either `_cli_parse_args` or `cli_parse_args` config.